src/adapter.js

```diff
@@ -47,6 +47,19 @@
             System.bundles = [];
         }
 
+        var systemInstantiate = System.instantiate;
```

Comment:
Could you add a comment somewhere in this new block of code explaining what it is doing/why? Looks like it replaces paths of the coverage files to include the `/base/` path?